stats/zvalues.h

```diff
@@ -2,19 +2,9 @@
 // Defines the z-values of features in the chart view.
 // Objects with higher z-values are painted on top of objects
 // with smaller z-values. For the same z-value objects are
-// drawn in order of addition to the graphics scene.
+// drawn in order of addition to the scene.
 #ifndef ZVALUES_H
 
-struct ZValues {
-	static constexpr double grid = -1.0;
-	static constexpr double series = 0.0;
```

Comment:
comment on the commit message... are those 'flot' values running fast? (sorry, bad German joke)